example/sl-prediction/train_and_predict.py

```diff
@@ -137,8 +145,11 @@ def train_and_predict(fold, _pairs_train, _X_train, _y_train, _pairs_test, _X_te
     # SVM
     elif args.classifier == 'svm':
         # Train the Linear SVM
-        clf = LinearSVC(C=args.svm_C, tol=args.svm_tolerance)
+        svc = LinearSVC(tol=args.svm_tolerance, random_state=_random_state+1)
+        clf = GridSearchCV(svc, dict(C=args.svm_Cs), cv=inner_cv,
```

Comment:
You can speed this up by giving `n_jobs` parameter to GridSearchCV and parallelize the training.